examples/asr/transcribe_speech.py

```diff
@@ -61,7 +61,10 @@ class TranscriptionConfig:
     # General configs
     output_filename: Optional[str] = None
     batch_size: int = 32
-    cuda: Optional[bool] = None  # will switch to cuda if available, defaults to CPU otherwise
+    # Set `cuda_device` to int to define CUDA device. If 'None', will look for CUDA
+    # device anyway, and do inference on CPU only if CUDA device is not found.
+    # If `cuda_device` is a negative number, inference will be on CPU only.
+    cuda_device: Optional[int] = None
```

Comment:
Pedantic, but can be revert the name cuda_device and just call if cuda.
Based to pytorch - ("cuda:0") in